raiden/transfer/state.py

```diff
@@ -960,6 +960,8 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'signature': serialization.serialize_bytes(self.signature),
             'sender': to_checksum_address(self.sender),
             'chain_id': self.chain_id,
+            # Makes the balance hash available to query
+            'balance_hash': serialize_bytes(self.balance_hash),
```

Comment:
Should this be made available for `BalanceProofUnsignedState` as well?

Reply:
Uhh, definitely, good catch

Reply:
done by a6f2acdb934125a82340420ea8166d18694ea755